README.md

```diff
@@ -16,6 +18,20 @@ gene_names = data.gene_names_at_locus(contig=6, position=29945884)
 exon_ids  = data.exon_ids_of_gene_name('HLA-A')
 ```
 
+# Installation
+
+You can install PyEnsembl using [pip](https://pip.pypa.io/en/latest/quickstart.html):
+
+```sh
+pip install pyensembl
+```
+
+This should also install any required packages, such as [datacache](https://github.com/hammerlab/datacache) and
+[pyfaidx](https://pypi.python.org/pypi/pyfaidx).
+
+The first time you use PyEnsembl, there will be a delay of several minutes while the library downloads and parses Ensembl data files.
```

Comment:
Plan for next update: `EnsemblRelease.download_annotations()`, `EnsemblRelease.download_transcript_sequences()`. What do you think?